pkg/pdfcpu/paperSize.go

```diff
@@ -49,14 +49,14 @@ func (d Dim) ToInches() Dim {
 	return Dim{d.Width * userSpaceToInch, d.Height * userSpaceToInch}
 }
 
-// ToCentimetres converts d to centimetres.
-func (d Dim) ToCentimetres() Dim {
+// ToCentimeters converts d to centimetres.
```

Comment:
I am ok with using the American spelling across the board for centimeters and millimeters, but this has to be done consistently for all occurences in the code base including all comments.

For function name changes please also fix all references eg. like in context.go

Reply:
Thanks for your comments.

I was looking into it and there are a lot of cases like that, from switch/case scenarios to func calls in many places. I could make the changes, but there's a huge risk of breaking too much of other people's code. From godoc.org it felt like an isolated thing, but in fact it's not (sorry for the lack of proper investigation).

Please feel free to reject the changes - the small bug fix is still valid though - or let me know if you still wanna change all occurrences.

Reply:
I am happy to merge this in if you limit the change to the obvious bugfix.
I am going to take care of the spelling issue later on myself.